ext/user_data/squisher/ethereum/StatusDesktop/nodekey

```diff
@@ -0,0 +1 @@
+2f37a1a935ae66e9f6bf77441cc2b611526687413c7784056b499a8c538dc734
```

Comment:
should `ext/user_data` be ignored when commiting? Why do we need that?

Reply:
same for archivedata, data, ethereum and other data folder data. Please remove from the commit

Reply:
I prepared user data with tokens in the wallet and used it in the test.
I can not generate tokens by API because all services require a captcha code.